### ISSUE TITLE
Dynamically Display Applicable Funding Opportunities on Oscar Dashboard

### ISSUE THREAD

**Issue by aaronvux** [newjersey/navigator.business.nj.gov#1362]
As Oscar, I need to have the content on my Navigator Dashboard display dynamically so that I get the most useful information.

## Acceptance criteria

**Display funding opportunity content dynamically based on the logic below:**
- Onboarding Home Based Business = yes: display Home Based Business opportunities from datasource with yes or don't know values.
- Onboarding Home Based Business = no: display all funding opportunities 
[look at column filter:  home based business]
*AND*
- Onboarding Location = [city value] : display any opportunity from datasource with "All" or the county where the city resides. [look at column filter: county]
*AND*
- Onboarding Industry Sector= [industry sector value]: display any opportunity from datasource with All or matching industries. [look at column filter: industry]
*AND*
- Onboarding # of employees = [# of employees = 0]: display any opportunity from datasource with # of employees = "n/a"
- Onboarding # of employees = [# of employees = >0]: display any opportunity from datasource with # of employees = "yes" and "n/a" [look at column filter: business size]
*AND*
- Don't display any opportunities from datasource that have been marked as do not publish. [look at column publish/stage/archive]

## Notes

- [ ] UX Review
- [ ] PO Approval

**Comment by aaronvux** [comment]
Removed number of employees requirement.

**Comment by aaronvux** [comment]
You can view the airtable here:
https://airtable.com/appMLFtbZCJrJPTf4/tblFkGCCfFEWORzDA/viw3t0KejenEqdqh0?blocks=hide

**Comment by Jesslax** [comment]
Should add acceptance criteria about the order opportunities display, such that:
1. certifications display at the top (For now that is just DVOB)
2. funding programs display in the following order:
- prioritize based on the input in the column "status," items marked "deadline" should display first, followed by "first-come-first-served," followed by "open"

**Comment by aaronvux** [comment]
@aloverso walked through a couple with @Jesslax.  I think we may need to move the archived funding opportunities into the correct folder for the logic to work. @kmelefsky let's talk about the implications of this at standup on Monday.

**Comment by aaronvux** [comment]
@aloverso I don't think this is correct.  Angel Investors Funding Opportunity is displaying even though I have 500 employees.  The limit looks to be 225 in Airtable.

https://user-images.githubusercontent.com/36163601/153964163-3c759f90-0a29-4dad-9a96-333b6f46a77b.mov

**Comment by Jesslax** [comment]
@aaronvux @aloverso , so from what I can tell, home-based business and county filters are working well. The industry sector filter is not working correctly, for example, I selected "digital and print media" as my sector, or when I selected "manufacturing" as my scetor, I saw the Angel Investor Tax Credit Program, which I should not see because my industry didn't match up. Filtering out items tagged as "do not published" is not working, because I see  the "Opportunity Partnership" program which is marked "do not publish." The filter for # of employees works sometimes but not all the time, from what I can tell it's not working for any program that DOL runs. For example, I marked 0 for my business size, but the funding program "Industry Partnership" which requires employees is displaying.
